frontend/src/sockets/countdown.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Function for processing game start countdown
+ * @param {*} socket
+ * @param {*} cb call back function
+ */
+export function countdown(socket, cb) {
+  socket.on('countdown', (count) => {
```

Comment:
an object is returned from the socket, so maybe use object destructuring? (e.g. ({count}) => {...})

Reply:
ah true, will do